short-read-mngs/idseq-dag/idseq_dag/steps/generate_alignment_viz.py

```diff
@@ -42,14 +41,15 @@ def run(self):
         db_type = "nt"  # Only NT supported for now
         # TODO: Design a way to map in/out files more robustly, e.g. by name/type
         annotated_m8 = self.input_files_local[0][0]
-        annotated_fasta = self.input_files_local[1][0]
+        annotated_nt_fasta = self.input_files_local[1][0]
+        annotated_nr_fasta = self.input_files_local[1][2]
```

Comment:
I'm not familiar with the structure of `input_files_local` is.Is the structure defined somewhere?

Reply:
It is a holdover from idseq-dag unfortunately